### PR TITLE
Skeleton stable

### PR DESCRIPTION
**Pull request by nonbhoward**
i propose using this skeleton as a starting point

### CODE REVIEW

**Comment by JohnKrause** [comment]
My primary feedback here is that I am not a fan of long library names personally
I am more of a:
from libs.scheduler import Scheduler
import libs.player as Player
ServerPlayer = Player.VLCStreamer()
ServerViewer = Player.VLCClient()
But this is just nitpicking ofcourse

**Comment by nonbhoward** [comment]
That is definitely my preference as well, shorter is always better as long as it doesn't impact readability.

[Tangent, except with function names where I would argue longer is always better, but just like in comedy, the only words that should be there are the absolute minimum for understanding. So, if you can't defend the need for a word in a phrase, or describe what it explains, remove it.]

Back on topic, the reason I do it this way is to prevent collision with import code, which has happened to me on a number of occasions where I tried to use minimalist names and ended up colliding with import code. The interpreter would find my code in path when it was looking for library code. So after a series of time-consuming bugs like that, I reformed.

There has to be some effort made to GUID your project structure or you'll end up with a variety of hard-to-diagnose bugs that come from the interpreter not knowing what you're trying to call.

TLDR, we can def change the names to anything, just keep uniqueness in mind. Vague names lead to headaches.

**Comment by nonbhoward** [comment]
That said, I am gonna drop this PR. I think it is too large for a collaborative project and PRs should probably be limited to module-level.
